rust/actyx/ax/src/cmd/events/publish.rs

```diff
@@ -24,13 +27,45 @@ use util::{
 pub struct PublishOpts {
     #[structopt(flatten)]
     console_opt: ConsoleOpt,
-    /// event payload (JSON)
-    payload: serde_json::Value,
-    #[structopt(long, short)]
+
+    /// Event payload, needs to be valid JSON.
```

Comment:
With the line break the explanation isn't caught by `structopt`.
This, like the other changes were stolen from same feature in settings

Reply:
With the line break structopt will only show the following part with `--help` (long help), not with `-h` (short help). I think the current text is too long for “short help”, this can be changed either using the line break mechanism or using dedicated `#[structopt(long_help = "...")]`.

Reply:
TIL but anyway I'm not a fan of either as with the line break it looks like:
```
ARGS:
    <NODE>       
            the IP address or <host>:<admin port> of the node to perform the operation on

    <payload>    
            Event payload, needs to be valid JSON.
            
            You may also pass a file in using the syntax `@file.json` or have the command read from standard input
            using `@-`.
```
But with the other option, the docs are the ones that suffer.

I guess I'll keep the line break

Reply:
How about shortening it?

    event payload (JSON) or @FILE for reading from a file (@- for stdin)